app/services/editions/publishing/EditionsBucket.scala

```diff
@@ -10,15 +10,13 @@ import PublishedIssueFormatters._
 class EditionsBucket(s3Client: AmazonS3, bucketName: String) {
   def createIssuePrefix(issue: PublishedIssue): String = s"${issue.name}/${issue.issueDate.toString}"
 
-  def createIssueFilename(issue: PublishedIssue): String = {
-    val keyname = issue.version.getOrElse("preview")
-    s"$keyname.json"
-  }
```

Comment:
you prefer the change the filename to `event-id` anyway ? we are passing that to object metadata now, what was the version before? the date ?

Reply:
I think It'll keep things simple to more to a guid everywhere we can.

Reply:
👍